client/wc-api/onboarding/constants.js

```diff
@@ -21,4 +21,8 @@ export const pluginNames = {
 		'WooCommerce ShipStation Gateway',
 		'woocommerce-admin'
 	),
+	'google-ads': __(
```

Comment:
This one needs to be updated as well or the plugin names won't work.

<img width="534" alt="Screen Shot 2020-03-12 at 4 09 38 PM" src="https://user-images.githubusercontent.com/10561050/76536064-20f6f800-647c-11ea-9266-2983461cb0bc.png">